operator/pkg/manifest/helm_processor.go

```diff
@@ -508,3 +517,11 @@ func (h *helm) InvalidateConfigAndRenderedManifest(deployInfo types.InstallInfo,
 	// new entry
 	return newHash, nil
 }
+
+func (h *helm) ToRESTConfig() (*rest.Config, error) {
```

Comment:
instead of casting the methods here from the clients struct, instead of passing that, you could just pass the clients into the operations imo. then helm/kustomize render src would not have to implement these 2 methods.

Reply:
We cannot do that with the current setup, since the clients are cached (helm & kustomize) and not operations. So to read the cluster info from the cache needs type casting. I combined these methods to a combined ClusterInfo type, which can directly be returned. This is exactly the part we try to unify :D

Reply:
I think my understanding is that the operations were cached before, not the clients. To read the cluster info, it just needs to get this info passed in the method imo.

Reply:
this implementation was changed.